scan.go

```diff
@@ -384,9 +384,14 @@ func resultData(ip, fs, ls string) (scanData, error) {
 		filter.Values = append(filter.Values, fmt.Sprintf("%%%s%%", ip))
 	}
 	if fs != "" {
-		t, _ := time.Parse(dateTime, fs)
-		filter.Where = append(filter.Where, `firstseen=?`)
-		filter.Values = append(filter.Values, t)
+		i, err := strconv.ParseInt(fs, 10, 0)
```

Comment:
This was the main UI bug. A missing conversion of fs into a UNIX time.

Reply:
Whoops! Good catch, thanks Nadin!